### ISSUE TITLE
mislabeled output images when using GUI

### ISSUE THREAD

**Issue by szkafander** [Heldenkombinat/Logodetect#37]
Apparently, some output images are mislabeled when using the GUI. For example:
![heineken-0-0-1547070960_output](https://user-images.githubusercontent.com/42718361/142797697-0efbcf04-5511-47dc-bd82-2dc172f4002c.jpg)

![3500_output](https://user-images.githubusercontent.com/42718361/142797726-4010b12f-3625-414a-8563-d3da0303e3e4.jpg)
.

**Comment by jorgedch** [comment]
Thank you for your feedback @szkafander.

Indeed, it is the case that the accuracy of the logo detections of the provided model is not very high. We describe in the README the following:

"additionally to sharing with you a couple of algorithms to get started with one-shot logo-detection, the aim of this project is to develop a flexible architecture to facilitate the comparison of different algorithms for one-shot object recognition." and "The models that we are including in the repo achieved a reasonable performance after a few training epochs. However, if you would like to improve their performance you can find pointers to various datasets in [6], which can be used in the training part of this project."

So the aim of the project is not to provide the user with a highly performant pre-trained model, but with a playground to allow a standardized comparison of one-shot algorithms. However, as we continue to refine the training of models for the detection of logos (and other classes of objects, such as traffic signs, faces, etc.), we are very happy to share those improved models later on with the community.

**Comment by szkafander** [comment]
> Thank you for your feedback @szkafander.
> 
> Indeed, it is the case that the accuracy of the logo detections of the provided model is not very high. We describe in the README the following:
> 
> "additionally to sharing with you a couple of algorithms to get started with one-shot logo-detection, the aim of this project is to develop a flexible architecture to facilitate the comparison of different algorithms for one-shot object recognition." and "The models that we are including in the repo achieved a reasonable performance after a few training epochs. However, if you would like to improve their performance you can find pointers to various datasets in [6], which can be used in the training part of this project."
> 
> So the aim of the project is not to provide the user with a highly performant pre-trained model, but with a playground to allow a standardized comparison of one-shot algorithms. However, as we continue to refine the training of models for the detection of logos (and other classes of objects, such as traffic signs, faces, etc.), we are very happy to share those improved models later on with the community.

Just FYI, all labels in the output images are "pepsi" (apparently irrespective of the data that I feed it), so I have a feeling that this might be a bug in labeling the images. Disregard this comment if you are sure that the inaccuracy is in the matcher model.

**Comment by jorgedch** [comment]
That's a very good observation, many thanks! I'll double check in case there is a bug in the labelling and image-annotation modules and update the repo accordingly 👍

**Comment by advaitian** [comment]
https://github.com/Heldenkombinat/Logodetect/blob/master/logodetect/constants.py

Line 36: CHOICE = 1  

change it to CHOICE = 0

**Comment by jorgedch** [comment]
Thanks for the suggestion @advaitian. CHOICE=1 should stay to speed up tests as the 50 ms video is the shortest one. The reported issue is related to the undertrained model provided as an example, as explained in https://github.com/Heldenkombinat/Logodetect/issues/37#issuecomment-976090564
